package.json

```diff
@@ -10,8 +10,8 @@
     "start": "node src/index.js",
     "start:live": "nodemon src/index.js",
     "db:build": "node src/model/database/db_build.js",
-    "lint": "./node_modules/.bin/eslint src/*.* src/**/*.js public/*.js",
-    "lint:fix": "./node_modules/.bin/eslint src/*.* src/**/*.js public/*.js --fix"
+    "lint": "./node_modules/.bin/eslint src/*.* src/**/*.js public/js/*.js",
+    "lint:fix": "./node_modules/.bin/eslint src/*.* src/**/*.js public/js/*.js --fix"
```

Comment:
maybe try changing this to:

```json
"lint": "./node_modules/.bin/eslint src/*.* src/**/*.js public/js/*.js",
"lint:fix": "npm run lint -- --fix",
```

would DRY the script a little

Reply:
I knew that you could do something like that, but I didn't know you had to add the extra -- so I couldn't get it to work. Thanks